pkg/controller/servicebinding/mapper.go

```diff
@@ -42,7 +42,7 @@ func isSecret(obj runtime.Object) bool {
 
 // isSBRService checks whether the given obj is a service in given sbr.
 func isSBRService(sbr *v1alpha1.ServiceBinding, obj runtime.Object) bool {
-	services := servicesOrEmptyList(sbr.Spec.Services)
+	services := sbr.Spec.Services
```

Comment:
do we really need `services` var? We can iterate directly `sbr.Spec.Services` in for loop?

Reply:
No, we don't need it, I'll update it.

Reply:
Done.